src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply";
+    public static final String BUY = "buy";
+    public static final String COMMA_SEPARATOR = ",";
+    public static final int TYPE_INDEX = 0;
+    public static final int VALUE_INDEX = 1;
+
+    private int supply = 0;
+    private int buy = 0;
+    private int result = 0;
```

Comment:
make them local variables instead

Reply:
I use them in a few methods: calculateStatistics and writeStatisticToFile. so I can't make them local :(